internal/v6provider/data_source_load_balancers.go

```diff
@@ -97,15 +97,15 @@ func (ds *LoadBalancersDataSource) Read(
 			return
 		}
 
-		if res.JSON200 == nil {
+		var totalPagesError error
+		totalPages, totalPagesError = res.JSON200.Pagination.TotalPages.Get()
+		if totalPagesError != nil {
 			resp.Diagnostics.AddError(
 				"Load Balancer List Error",
-				"nil response")
+				totalPagesError.Error())
```

Comment:
suggestion: Same here regarding total pages `.Get()` error mentioned in previous comment.